### PR TITLE
Playlab page

### PR DESCRIPTION
**Pull request by caleybrock**
screen shot 2015-11-19 at 9 48 09 am

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/playlab.haml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Play Lab
+theme: responsive
+social:
+  "twitter:card": photo
+  "twitter:site": "@codeorg"
+  "twitter:url": "http://code.org/playlab"
+  "twitter:image:src" : "https://code.org/images/mc/mc_social.jpg"
+  "twitter:title": "Minecraft Hour of Code"
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/playlab.haml]: This page will have Minecraft social tags for Twitter. Best to remove everything under social.

File: pegasus/sites.v3/code.org/public/playlab.haml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  "og:image" : "https://code.org/images/mc/mc_social.jpg"
+  "og:image:width": "1920"
+  "og:image:height": "1080"
+  "og:video": ""
+  "og:video:width": ""
+  "og:video:height": ""
+  "og:video:type": ""
+---
+
+- page_mode = PageMode.get(request)
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/playlab.haml]: Remove if we're not using

File: pegasus/sites.v3/code.org/public/playlab.haml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      %div.choice
+        %div.choice-container
+          %div.choice-label.purple= I18n.t(:playlab_classic_title)
+          %img.character-img{src: "/images/playlab_original3.png"}
+    %a{href: "https://studio.code.org/s/gumball/stage/1/puzzle/1"}
+      %div.choice
+        %div.choice-container
+          %div.choice-label.blue= I18n.t(:playlab_gumball_title)
+          %img.character-img{src: "/images/playlab_gumball.png"}
+    %a{}
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/playlab.haml:39]: no link?

File: pegasus/sites.v3/code.org/public/playlab.haml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+- page_mode = PageMode.get(request)
+
+%link{href: "/css/tutorial-landing.css", rel: "stylesheet"}
+
+%div{style: "clear:both"}
+
+.playlab-container
+  %h1= I18n.t(:playlab_choose_theme)
+  %div.image-collection
+    %a.while-link{href: "https://studio.code.org/s/playlab/stage/1/puzzle/1"}
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/playlab.haml:29]: What's the flow to this page? Should these link be the /api links instead? If not the /api link, it should at least be the /reset links.

File: pegasus/sites.v3/code.org/public/playlab.haml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+      %div.choice
+        %div.choice-container
+          %div.choice-label.purple= I18n.t(:playlab_classic_title)
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/playlab.haml:32]: If you added new strings, don't forget to

`cd i18n/code.org`
`./sync-pegasus.sh`
`commit all locale files in pegasus/cache/i18n`

File: pegasus/sites.v3/code.org/public/playlab.haml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+          %img.character-img{src: "/images/playlab_gumball.png"}
+    %a{}
+      %div.choice
+        %div.choice-container
+          %div.choice-label.green= I18n.t(:playlab_iceage_title)
+          %img.character-img{src: "/images/playlab_iceage.png"}
+
+%div{style: "clear:both"}
+
+.inspire-videos{style: "clear: both;"}
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/playlab.haml:47]: is `clear: both` twice in 47 and 49 necessary?

**Comment by tanyaparker** [comment]
Added my notes

File: pegasus/sites.v3/code.org/public/playlab.haml (hunk 3 above)
Comment (caleybrock) [inline pegasus/sites.v3/code.org/public/playlab.haml:39]: its "coming soon" until we get approval

File: pegasus/sites.v3/code.org/public/playlab.haml (hunk 4 above)
Comment (caleybrock) [inline pegasus/sites.v3/code.org/public/playlab.haml:29]: for now I'll do /reset, and then I'll follow up. I want to make sure those api links work.

File: pegasus/sites.v3/code.org/public/playlab.haml (hunk 5 above)
Comment (caleybrock) [inline pegasus/sites.v3/code.org/public/playlab.haml:32]: already done

**Pull request merged**